src/Angor/Avalonia/AngorApp/Composition/SectionsFactory.cs

```diff
@@ -0,0 +1,40 @@
+using AngorApp.Core;
+using AngorApp.Sections;
+using AngorApp.Sections.Founder;
+using AngorApp.Sections.Home;
+using AngorApp.Sections.Portfolio;
+using AngorApp.Sections.Shell;
+using AngorApp.Sections.Shell.Sections;
+using AngorApp.Sections.Wallet;
+using AngorApp.UI.Services;
+using Microsoft.Extensions.DependencyInjection;
+using Separator = AngorApp.Sections.Shell.Sections.Separator;
+
+namespace AngorApp.Composition;
+
+public class SectionsFactory(IServiceProvider serviceProvider) : ISectionsFactory
+{
```

Comment:
what are sections exactly? is this like sections of the UI?

Reply:
These are the sections that appear in the Sidebar. Their have a ViewModel as content, and they are lazily loaded as needed.